[2022-03-17 15:12:18,081] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_example_dag.task_group_Dimension_table.load_users_dim_table manual__2022-03-17T07:12:01.605330+00:00 [queued]>
[2022-03-17 15:12:18,094] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_example_dag.task_group_Dimension_table.load_users_dim_table manual__2022-03-17T07:12:01.605330+00:00 [queued]>
[2022-03-17 15:12:18,094] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 15:12:18,095] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-03-17 15:12:18,095] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 15:12:18,104] {taskinstance.py:1264} INFO - Executing <Task(LoadDimensionOperator): task_group_Dimension_table.load_users_dim_table> on 2022-03-17 07:12:01.605330+00:00
[2022-03-17 15:12:18,106] {standard_task_runner.py:52} INFO - Started process 43988 to run task
[2022-03-17 15:12:18,116] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'udac_example_dag', 'task_group_Dimension_table.load_users_dim_table', 'manual__2022-03-17T07:12:01.605330+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/udac_example_dag.py', '--cfg-path', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpsddzzse2', '--error-file', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpb1qsk2da']
[2022-03-17 15:12:18,118] {standard_task_runner.py:77} INFO - Job 64: Subtask task_group_Dimension_table.load_users_dim_table
[2022-03-17 15:12:18,209] {logging_mixin.py:109} INFO - Running <TaskInstance: udac_example_dag.task_group_Dimension_table.load_users_dim_table manual__2022-03-17T07:12:01.605330+00:00 [running]> on host oneforall-nickdemacbook-pro.local
[2022-03-17 15:12:18,283] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=udac_example_dag
AIRFLOW_CTX_TASK_ID=task_group_Dimension_table.load_users_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T07:12:01.605330+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-17T07:12:01.605330+00:00
[2022-03-17 15:12:18,284] {load_dimension.py:83} INFO - Starting to truncate users
[2022-03-17 15:12:18,295] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-17 15:12:19,568] {dbapi.py:225} INFO - Running statement: 
        TRUNCATE TABLE "public"."users"
        , parameters: None
[2022-03-17 15:12:20,250] {load_dimension.py:93} INFO - Starting to insert users
[2022-03-17 15:12:20,253] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-17 15:12:20,254] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-03-17 15:12:20,255] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-03-17 15:12:20,255] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-03-17 15:12:20,256] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,257] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,258] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,259] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-03-17 15:12:20,260] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,260] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,261] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,262] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 15:12:20,269] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-17 15:12:21,920] {dbapi.py:225} INFO - Running statement: 
        INSERT INTO "public"."users" (userid, first_name, last_name, gender,level)
 
        SELECT distinct userid, firstname, lastname, gender, level
        FROM staging_events
        WHERE page='NextSong'
    
        , parameters: None
[2022-03-17 15:12:23,260] {dbapi.py:233} INFO - Rows affected: 104
[2022-03-17 15:12:24,268] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=udac_example_dag, task_id=task_group_Dimension_table.load_users_dim_table, execution_date=20220317T071201, start_date=20220317T071218, end_date=20220317T071224
[2022-03-17 15:12:24,284] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-17 15:12:24,320] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
